x/programs/runtime/import_program_test.go

```diff
@@ -151,3 +151,20 @@ func TestImportGetRemainingFuel(t *testing.T) {
 	require.NoError(err)
 	require.LessOrEqual(into[uint64](result), program.Runtime.DefaultGas)
 }
+
+func TestDeferredBytes(t *testing.T) {
+	require := require.New(t)
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	program := newTestProgram(ctx, "call_program")
+
+	expected, err := serialize(int64(0))
+	require.NoError(err)
+
+	result, err := program.Call("call_deferred",
+		program.Address, uint64(1000000))
+	require.NoError(err)
+	require.Equal(expected, result)
+}
```

Comment:
This is simply a convenience type that *should* make it easier to write the multi-sig. It should have a unit test, but adding integration tests like this is overkill.

Reply:
Got it. I added 3 unit tests that shows the usage of `DeferDeserialize` compared to a borsh deserialization.